### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by SlaviaF**

### CODE REVIEW

File: javascript/javascript1/week2/school.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+console.log(class07Students);
+
+function existingStudent(student) {
+    for (i = 0; i < class07Students.length; i++) {
+        if (class07Students[i] === student) {
+            return true;
+        }
+
```
Comment (flom84) [inline javascript/javascript1/week2/school.js]: You might consider returning false when there is no students.
```suggestion
return false
```

File: javascript/javascript1/week2/school.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName === "Queen") {
```
Comment (flom84) [inline javascript/javascript1/week2/school.js]: With your current _if_ statements the queen is not added to the array

File: javascript/javascript1/week2/candy.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+//CAN YOU BUY MORE
+
+let amountToSpend = Math.random() * 100;
+function canBuyMoreCandy() {
+    let totalPriceOfCandy = 0;
+    for (i = 0; i < boughtCandyPrices.length; i++) {
+        totalPriceOfCandy = totalPriceOfCandy + boughtCandyPrices[i];
+
+    }
+    if (totalPriceOfCandy > amountToSpend) {
```
Comment (flom84) [inline javascript/javascript1/week2/candy.js:28]: I think if you have more money then you can buy more
```suggestion
    if (totalPriceOfCandy < amountToSpend) {
```

File: javascript/javascript1/week2/school.js (hunk 1 above)
Comment (SlaviaF) [reply javascript/javascript1/week2/school.js]: Yes ...I did it now but I was little confused about it before as when I was adding else statements as false it was returning me the opposite value, but returning the function with true gave me the correct answer

File: javascript/javascript1/week2/candy.js (hunk 3 above)
Comment (SlaviaF) [reply javascript/javascript1/week2/candy.js:28]: Ohhh yes..Just noticed it

**Pull request merged**